### PR TITLE
test: fix data import test failure

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: apps/emqx_management/test/emqx_mgmt_SUITE.erl [hunk 1]
```diff
@@ -397,3 +394,11 @@ mock_print() ->
 
 unmock_print() ->
     meck:unload(emqx_ctl).
+
+safe_unmeck(Module) ->
```
Comment (thalesmg) [inline apps/emqx_management/test/emqx_mgmt_SUITE.erl:398]: For my own curiosity: why was this needed?

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_management/test/emqx_mgmt_SUITE.erl (hunk 1 above)
Comment (zmstone) [reply apps/emqx_management/test/emqx_mgmt_SUITE.erl:398]: now that I look at it again. this should work `try meck:unload(module) catch _ : _ -> ok end`
it's a direct exit, but not a 'EXIT' signal.

**Review by HJianBo — APPROVED** [review]
